app/recordtransfer/templates/recordtransfer/email/email_base.html

```diff
@@ -0,0 +1,102 @@
+{% load i18n %}
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta name="description"
+              content="Email communication from the National Centre for Truth and Reconciliation.">
+        <title>{% blocktrans %}Email from Our Company{% endblocktrans %}</title>
```

Comment:
Can you set a title inside each email that extends the base like this?

```suggestion
        <title>{% block metatitle %}{% endblock metatitle %}</title>
```

Reply:
Resolved by https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/575/commits/9056ebb201c736005555bf95b02b3097d826b5a6. I just removed the title from the email. It wold give me an error that I should keep the title tag still but its empty